### DISCUSSION TITLE
Error when saving advanced server settings

### DISCUSSION THREAD

**Discussion by wmurphyrd** [mozilla/hubs#3334] [Hubs Cloud Questions] [answered]
I'm no longer able to save settings on the admin->server settings->advanced tab. I receive an error popup when clicking save, even if I haven't made any changes. If I do make changes, they are lost if I click save and then refresh after seeing the error. 

Error logs are here: https://gist.github.com/wmurphyrd/73352dc1e00e7e14a94f44cecc8f4e93

Details:

* I have a digital ocean hubs cloud.
* The last successful change I made was when I enabled CloudFlare workers.
* Troubleshooting steps I have tried: undeploying my custom client.

**Comment by robinkwilson** [comment, marked as answer]
I would restart your instance if you can. If you make an issue in https://github.com/mozilla/hubs-cloud/issues it would help us look! Especially with your repro steps! Check your console logs + network tab would also help.

Also unfortunately, we're unable to address DigitalOcean bugs as quickly as we can AWS since it's in alpha and most support is going towards AWS offering so you're aware. Not ideal I understand.

**Reply by robinkwilson** [reply]
Marking as answered for now unmark when there's more info or an issue/repro link!
